schema.go

```diff
@@ -8,30 +8,30 @@ import (
 // Schema reflects a schema from a Go value.
 //
 // A schema is itself HCL.
-func Schema(v interface{}) (*AST, error) {
-	ast, err := marshalToAST(v, true)
+func Schema(v interface{}, opt *MarshalOptions) (*AST, error) {
```

Comment:
This should be `options...MarshalOption`.

Reply:
Done